### PR TITLE
[RAPPS-DB] Adding missing icons for Audio, Video, Graphics and 15 Gaming apps.

### PR DESCRIPTION
**Pull request by Gathub22**
I'm adding missing icons to existing apps to improve readability and overall RAPPS' appearance.

I only add icons designed to the app itself or their developers' icons. I'm trying to make all icons transparent so in case the RAPPS' theme changes, it doesn't have ugly white backgrounds. Also I'm trying to add the exact icons that were used at the time these applications were released. Ofc those icons are completely owned by their respective owners, not by me in any way.

I'm adding icons based on these guidelines:

1. The icon is used <ins>to represent the game itself</ins> or its developers.
2. The icon is used <ins>to represent the software itself and not its origin</ins>. For example, if the software is a PC port of an old Nintendo game, I won't add the original logo of that ancient game to RAPPS. Only if the port has its own.
3. The icon <ins>should be the closest to the one that relates to the version of the software</ins>. For example, it would be really innacurate to use Microsoft's 2012 logo to represent a Windows XP application.
4. The icon will be 100x100 px.
5. The icon will be fully transparent.

[There is an ongoing Jira issue to discuss about this problem.](https://jira.reactos.org/browse/CORE-19614)

I'm open for any suggestions or help :)   

## Completed categories

- [x] Audio 🎵
- [x] Video 🎬
- [x] Graphics 🖌️
- Games and fun 🕹️ (15 new icons)

In total 32 new icons have been added to this PR.

## In future PRs
- Internet and network 🌐
- Office 🖊️
- Development 🦺
- Edutainment 📚
- Engineering ⚙️
- Finance 💰
- Science 🧪
- Tools 🔧
- Drivers 💿
- Libraries 📦
- Themes 🎨
- Other 🗃️

### CODE REVIEW

**Comment by learn-more** [comment]
@Gathub22 what is the status of this PR?

**Comment by Gathub22** [comment]
@learn-more Bc of lack of time its stuck. However we could split this PR and prepare merging whats already done

**Comment by learn-more** [comment]
> @learn-more Bc of lack of time its stuck. However we could split this PR and prepare merging whats already done

That seems like the way forward.

File: GrafX2.txt [hunk 1]
```diff
@@ -8,6 +8,7 @@ URLSite = http://grafx2.chez.com/
 URLDownload = https://gitlab.com/GrafX2/grafX2/-/jobs/3140386057/artifacts/raw/install/grafx2-win32-2.8.3137.win32.exe
 SHA1 = df7ec1ee27dd509c484a277aa33546adc3616817
 SizeBytes = 2198528
+Icon = GrafX2.ico
```
Comment (learn-more) [inline GrafX2.txt:11]: If the icon has the same name as the `.txt` file, then you don't need this entry.

Comment (Gathub22) [reply GrafX2.txt]: Done :)  [5bc72ee](https://github.com/reactos/rapps-db/pull/240/commits/5bc72ee8f2742c831528e70fa38ec089223baec6)

Comment (Gathub22) [reply GrafX2.txt]: Okay they are all removed

**Review by learn-more — APPROVED** [review]

**Comment by julenuri** [comment]
I will check locally, and if everything works nicely, I will commit it. :smile:

File: icons/ballonfight.ico
Comment (julenuri) [inline icons/ballonfight.ico:1]: Typo mistake: Please, change the file name from ballonfight.ico to balloonfight.ico
Thanks.

**Review by julenuri — CHANGES_REQUESTED** [review]

**Review by julenuri — APPROVED** [review]

**Pull request merged**